{"Functions": {"MalformedChallengeException(java.lang.String, java.lang.Throwable)": {"Returns": [], "Parameters": [["@B_java#lang#String_E@", "the exception detail message"], ["@B_java#lang#Throwable_E@", "the Throwable that caused this exception, or null if the cause is unavailable, unknown, or not a Throwable"]], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Creates a new MalformedChallengeException with the specified detail message and cause.", "history": "Added in API level 1", "FullName": "public MalformedChallengeException (String message, Throwable cause)"}, "MalformedChallengeException()": {"Returns": [], "Parameters": [], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Creates a new MalformedChallengeException with a", "history": "Added in API level 1", "FullName": "public MalformedChallengeException ()"}, "MalformedChallengeException(java.lang.String)": {"Returns": [], "Parameters": [["@B_java#lang#String_E@", "the exception detail message"]], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Creates a new MalformedChallengeException with the specified message.", "history": "Added in API level 1", "FullName": "public MalformedChallengeException (String message)"}}, "Inheritance": ["@B_java#lang#Throwable_E@", "@B_java#lang#Exception_E@", "@B_org#apache#http#HttpException_E@", "@B_org#apache#http#ProtocolException_E@"], "ClassName": "org.apache.http.auth.MalformedChallengeException", "ClassDesc": "Please use @B_java#net#URL#openConnection_E@ instead. Please visit Signals that authentication challenge is in some way invalid or illegal in the given context"}